### DISCUSSION TITLE
Colors in the output are not displayed on Windows

### DISCUSSION THREAD

**Discussion by hack-Red-moon** [projectdiscovery/nuclei#1509] [Q&A] [answered]
Windows 10 terminal output command cannot display color normally

![image](https://user-images.githubusercontent.com/97647553/149896616-88347905-d637-4ee0-8463-15d1ecf4ad89.png)

**Comment by No-Github** [comment, marked as answer]
windows终端不支持类似 \033[31m 的显示，建议使用 -nc 关闭颜色输出

**Reply by hack-Red-moon** [reply]
3Q

**Comment by forgedhallpass** [comment]
Relevant documentation on limitations of the underlying coloring library we use, in case of Windows: https://github.com/logrusorgru/aurora#windows

There are workarounds on how to display colored output on Windows:
- enable ANSI Terminal Control through the Windows Registry
- use an alternative terminal emulator like: `MinTTY`(used by `Cygwin` and `Git Bash`), [Microsoft Terminal](https://github.com/microsoft/terminal) etc.

See the relevant superuser discussion: https://superuser.com/questions/413073/windows-console-with-ansi-colors-handling

Alternatively, as @No-Github mentioned, you can disable the coloring, using the `-nc` or `-no-color` flags.
